provider/go.mod

```diff
@@ -38,6 +38,11 @@ require (
 	google.golang.org/protobuf v1.35.1
 )
 
+require (
+	github.com/pulumi/pulumi/pkg/v3 v3.134.0
+	github.com/pulumi/pulumi/sdk/v3 v3.134.0
+)
+
```

Comment:
Your local ref was pointing at https://github.com/pulumi/pulumi-aws-native/pull/1844 instead of https://github.com/pulumi/pulumi-aws-native/pull/1852. I'll rebase you and look into making the branch name more random to prevent this.